bootstrap.sh

```
@@ -1,20 +1,32 @@
 #!/bin/bash
+
+echo "Creating cluster..."
+kind create cluster --config cluster.yml
```

Before calling `kind create cluster --config cluster.yml` ensure `cluster.yml` exists and is the expected file. If it's absent, fail early with a helpful message. Example check:

```bash
if [ ! -f cluster.yml ]; then echo "cluster.yml not found in repo root"; exit 1; fi
```

This ensures the technical constraint (“kind cluster must be created from a cluster.yml file”) is actually satisfied at runtime.